create-a-theme-checklist.js

```diff
@@ -30,69 +30,71 @@ async function createMilestone() {
 					'Automatically generated milestone for ' + theme + '.',
 			}
 		);
-	} catch ( error ) {
-		console.error( '\x1b[41m', 'Milestone already created.' );
+	} catch (error) {
+		console.error('\x1b[41m', 'Milestone already created.');
 	}
 }
 
-async function createIssues( milestoneNumber ) {
+async function createIssues(milestoneNumber) {
 	const issues = [
-		'Create base theme from boilerplate',
+		'Create base theme',
 		'Create a demo site on dotcom',
 		'theme.json: Typography settings',
 		'theme.json: Color palette',
 		'theme.json: Alternative color palettes',
 		'theme.json: Margin/spacing settings + content layout',
 		'Templates: Page templates - Index',
+		'Templates: Page templates - Default',
 		'Templates: Page templates - Search',
 		'Templates: Page templates - 404',
 		'Templates: Page templates - Archive',
-		'Templates: Header template part (including mobile)',
-		'Templates: Footer template part (including mobile)',
-		'Block patterns (if needed)',
-		'Comment form styles (dotcom and dotorg) (if needed)',
-		'Navigation (if needed)',
-		'Core block settings (if needed)',
+		'Templates: Post templates - Single',
+		'Templates: Header template part',
+		'Templates: Footer template part',
+		'Block patterns',
+		'Comment form styles (dotcom and dotorg)',
+		'Navigation',
```

Comment:
We probably don't need this, since it's included in the header template part issue